Multiple/Dual Frequency Oscillator/README.md

```diff
@@ -60,8 +93,8 @@ Connect the DFO pins to the corresponding points on your specific motherboard ve
 - [Installation guide for Playstation](https://www.consolesunleashed.com/guides/sony-playstation-dual-frequency-oscillator-install-guide/)
```

Comment:
maybe change "Installation guide for Playstation" to "Programming & Installation guide for Playstation"

Reply:
The website doesn't tell you how to program the DFO.